karate-core/pom.xml

```diff
@@ -13,6 +13,7 @@
     <properties>
         <antlr.version>4.9.2</antlr.version>
         <graal.version>21.2.0</graal.version>
+        <cucumber.reporting.version>5.3.1</cucumber.reporting.version>
```

Comment:
kindly move this into `karate-demo` because I'd like to keep `karate-core` lightweight

Reply:
also please submit PRs against `develop`